pipelines/performance-tests-pipeline.yml

```
@@ -680,6 +683,117 @@ jobs:
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: eu.gcr.io/census-gcr/gcloud-kubectl
+      params:
+        SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+        KUBERNETES_CLUSTER: ((performance-kubernetes-cluster-name))
+        GCP_PROJECT_NAME: ((performance-gcp-project-name))
+        CASE_PROCESSOR_REPLICAS: ((case-processor-replicas))
+        UAC_QID_REPLICAS: ((uac-qid-replicas))
+      run:
+        path: bash
+        args:
+          - -exc
+          - |
+            cat >~/gcloud-service-key.json <<EOL
+            $SERVICE_ACCOUNT_JSON
+            EOL
+
+            # Use gcloud service account to configure kubectl
+            gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+            gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
+
+            # Scale apps down
+            kubectl scale deployment action-scheduler --replicas=0
+            kubectl scale deployment case-api --replicas=0
+            kubectl scale deployment case-processor --replicas=0
+            kubectl scale deployment uacqidservice --replicas=0
+  on_failure: *slack_performance_setup_failure
+
+- name: "Wipe And Rebuild From Ground Zero"
+  disable_manual_trigger: true
+  serial: true
+  serial_groups: [
+    scale-down,
+    scale-apps,
+    action-scheduler,
+    case-api,
+    case-processor,
+    fieldwork-adapter,
+    notify-processor,
+    uac-qid-service,
+    pubsubsvc,
+    print-file-service,
+    exception-manager,
+    toolbox,
+    event-latency-monitor,
+    rabbitmonitor]
+  plan:
+  - get: every-minute
+    trigger: true
+    passed: ["Scale Down Pre Ground Zero"]
+  - task: "Wipe DB and rebuild from ground zero"
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: eu.gcr.io/census-gcr/gcloud-kubectl
+      params:
+        SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+        KUBERNETES_CLUSTER: ((performance-kubernetes-cluster-name))
+        GCP_PROJECT_NAME: ((performance-gcp-project-name))
+        CASE_PROCESSOR_REPLICAS: ((case-processor-replicas))
+        UAC_QID_REPLICAS: ((uac-qid-replicas))
+      run:
+        path: bash
+        args:
+          - -exc
+          - |
+            cat >~/gcloud-service-key.json <<EOL
+            $SERVICE_ACCOUNT_JSON
+            EOL
+
+            # Use gcloud service account to configure kubectl
+            gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+            gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
+
+            # Wipe and rebuild database
+            kubectl exec `kubectl get pods -o name | grep -m1 census-rm-toolbox | cut -d'/' -f 2` -- /bin/bash /app/groundzero/rebuild_from_ground_zero.sh
```

```suggestion
            kubectl exec $(kubectl get pods --selector=app=census-rm-toolbox -o jsonpath='{.items[*].metadata.name}') -- /bin/bash /app/groundzero/rebuild_from_ground_zero.sh
```
Can get the pod name directly from a `kubectl` command